assets/src/edit-story/components/panels/layer/panel.js

```diff
@@ -32,7 +32,7 @@ import { DEFAULT_LAYERS_VISIBLE, LAYER_HEIGHT } from './constants';
 import Layer from './layer';
 import useLayers from './useLayers';
 
-const LayerList = styled.div`
+const LayerList = styled.div.attrs({ role: 'listbox' })`
```

Comment:
With a `listbox` with `option`'s, this now works really well in announcing the current selection while preserving an interactive role for the buttons.

Reply:
It works very well now indeed!